test/e2e/scenario/rum/init.scenario.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-return */
+import { createTest, bundleSetup } from '../../lib/framework'
+import { asyncSetup } from '../../lib/framework/pageSetups'
+import { flushEvents } from '../../lib/helpers/sdk'
+
+describe('before init API calls', () => {
+  createTest('should be associated to corresponding views')
+    .withRum({ enableExperimentalFeatures: ['view-renaming'] })
+    .withRumInit((options) => {
+      ;(window.DD_RUM as any).addError('before manual view')
+      ;(window.DD_RUM as any).addAction('before manual view')
+      ;(window.DD_RUM as any).addTiming('before manual view')
+
+      setTimeout(() => (window.DD_RUM as any).startView('manual view'), 10)
+
+      setTimeout(() => {
+        ;(window.DD_RUM as any).addError('after manual view')
+        ;(window.DD_RUM as any).addAction('after manual view')
+        ;(window.DD_RUM as any).addTiming('after manual view')
+      }, 20)
+
+      setTimeout(() => (window.DD_RUM as any).init(options), 30)
+    })
+    .withSetup(bundleSetup, 'bundle')
+    .withSetup(asyncSetup, 'async')
```

Comment:
It could be nice to have somithing working with all setups. In the NPM setup, what about having something like `window.RUM_INIT = ${options.rumInit.toString()}`, that would be called from the app if it exists?